packages/ma-design-tokens/style-dictionary.mjs

```diff
@@ -2,15 +2,30 @@ import { register } from '@tokens-studio/sd-transforms';
 import StyleDictionary from 'style-dictionary';
 import { typeDtcgDelegate } from 'style-dictionary/utils';
 import { readFile } from 'node:fs/promises';
-import { createConfig } from '../../style-dictionary-config.js';
+import {
+  createConfig,
+  colorSchemeDarkPreprocessor,
+  colorSchemeDefaultPreprocessor,
+} from '../../style-dictionary-config.js';
 
 const build = async () => {
   const themeConfig = JSON.parse(await readFile('./src/config.json', 'utf-8'));
+
   StyleDictionary.registerPreprocessor({
     name: 'dtcg-delegate',
     preprocessor: typeDtcgDelegate,
   });
 
+  StyleDictionary.registerPreprocessor({
```

Comment:
Ik zou deze en L verplaatsen naar de preprocessor. Volgens [de SD docs](https://styledictionary.com/reference/hooks/preprocessors/#preprocessor-structure) is een preprocessor een object met een `name` en een `preprocessor` property.